### PR TITLE
wip: spanner: move admin clients to GAPIC stub

### PR DESCRIPTION
**Pull request by pongad**
Many integration tests are failing due to dropDatabase not being
implemented. This should be fixed in #3043.

### CODE REVIEW

**Comment by pongad** [comment]
All tests (including integrations) are passing now. PTAL

**Comment by hzyi-google** [comment]
LGTM

File: google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerImpl.java [hunk 1]
```diff
@@ -146,16 +147,23 @@
   @GuardedBy("this")
   private boolean spannerIsClosed = false;
 
-  SpannerImpl(SpannerRpc rpc, int defaultPrefetchChunks, SpannerOptions options) {
+  SpannerImpl(
+      SpannerRpc rpc, SpannerRpc gapicRpc, int defaultPrefetchChunks, SpannerOptions options) {
     super(options);
     this.rpc = rpc;
+    this.gapicRpc = gapicRpc;
```
Comment (garrettjonesgoogle) [inline google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerImpl.java]: Could you rename the existing `rpc` to something like `rawGrpcRpc` to make the contrast clearer?

Comment (pongad) [reply google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerImpl.java]: Done

**Comment by pongad** [comment]
@garrettjonesgoogle PTAL

**Pull request merged**